exercises/matrix/src/main/java/Matrix.java

```diff
@@ -0,0 +1,24 @@
+import java.util.regex.Pattern;
+
+public class Matrix {
```

Comment:
The class and the methods can all be [`package private`](https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html). It's best practice to be as restrictive as possible with access modifiers :)